inject/src/main/java/io/micronaut/context/converters/StringToCharArray.java

```diff
@@ -0,0 +1,16 @@
+package io.micronaut.context.converters;
+
+import io.micronaut.core.convert.ConversionContext;
+import io.micronaut.core.convert.TypeConverter;
+
+import javax.inject.Singleton;
+import java.util.Optional;
+
+@Singleton // #2614
+public class StringToCharArray implements TypeConverter<String,char[]> {
```

Comment:
Instead of a bean just for this one converter please add this to one of the existing `TypeConverterRegistrar` beans